src/vs/workbench/contrib/preferences/browser/settingsLayout.ts

```diff
@@ -17,18 +17,17 @@ export interface ITOCEntry<T> {
 }
 
 const defaultCommonlyUsedSettings: string[] = [
```

Comment:
Wait how does this work with the distro set?

Reply:
It's a fallback for OSS.